webapp/src/pages/LandingPage/LandingPage.css

```
@@ -3,15 +3,15 @@
   padding: var(--padding-top-bottom) var(--padding-sides);
   height: 100vh;
   min-height: 40rem;
-  overflow: scroll;
+  overflow: hidden;
   background-color: var(--color-background);
   color: var(--color-light);
   display: grid;
   gap: 2rem var(--column-gap);
   grid-template-columns: repeat(var(--column-number), 1fr);
   grid-template-rows: auto 1fr auto auto;
   grid-template-areas:
-    'title title title title'
+    'title title ... ...'
     'animation animation animation animation'
```

If we center the elements I would suggest to limit the animation width as:
```suggestion
    '... animation animation ...'
```
But I am happy with both solutions

---

I've tested it but we ended up with a slow animation. That being said I've fixed the animation width because it was overloading the grid container (I think you wanted to fix this in a first place)